js/chatbot/static/ChatBot.svelte

```diff
@@ -332,8 +332,10 @@
 		background-color: var(--chatbot-code-background-color);
 		padding: var(--spacing-xl) 10px;
 		direction: ltr;
+		white-space: no-wrap;
```

Comment:
This looks good to me! I don't know if it's possible but can we pin the scroll bar to the bottom of the viewable area of the code block?

It's a bit annoying to have to scroll up and down to be able to get at the horizontal scroll bar.
![chatbot_overflow](https://github.com/gradio-app/gradio/assets/41651716/589a478f-93ab-41db-b3cd-dbe91ffb2a9c)

I think it would be nice but I wouldn't want to hold this up if it's gonna take a long time.

Reply:
Sounds good!